kouta-backend/src/main/scala/fi/oph/kouta/client/KoodistoClient.scala

```diff
@@ -41,9 +40,12 @@ object KoodistoUtils {
     }
   }
 
-  def kooriUriToString(koodiUri: KoodiUri): String =
+  def koodiUriToString(koodiUri: KoodiUri): String =
     s"${koodiUri.koodiUri}#${koodiUri.latestVersio}"
 
+  def koodiUriStringsMatch(a: String, b: String): Boolean =
+    koodiUriFromString(a).koodiUri == koodiUriFromString(b).koodiUri
```

Comment:
Javassa ja Scalassa kannattaa vertailu tehdä näin:
`koodiUriFromString(a).koodiUri.equals(koodiUriFromString(b).koodiUri)`